app.rb

```diff
@@ -0,0 +1,31 @@
+require 'sinatra/base'
+require './lib/peep'
+require './lib/user'
+require './database_connection_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    @user = User.find(session[:user_id])
+    @peeps = Peep.all
+    erb(:index)
+  end
+
+  post '/new-peep' do
```

Comment:
I think it would be better to use '/peeps/new' route or similar that is RESTful (see code review doc).